Core/clim-core/commands/tables.lisp

```diff
@@ -344,7 +345,8 @@ designator) inherits menu items."
             (remhash (command-item-name item) (command-line-names table)))
           (remhash command-name (commands table))))))
 
-(defun map-over-command-table-menu-items (function command-table)
+(defun map-over-command-table-menu-items
```

Comment:
For consistency with other table mapping operators
- map-over-command-table-names
- map-over-command-commands
- map-over-command-table-translators

adding this keyword costs us nothing and it is backward compatible. it allows to access keystrokes and menu items that are specific to this command table (not inherited), or access all including inherited entries.

Reply:
I didn't get that from the commit message. Maybe

> According to the specification, most command table mapping operators accept a keyword INHERITED defaulting to T. Thanks to that it was possible to inhibit inherited objects mapping. 
The functions MAP-OVER-COMMAND-TABLE-KEYSTROKES and MAP-OVER-COMMAND-TABLE-MENU-ITEMS are exceptional in the specification because the keyword parameter is not specified for them.
As a McCLIM extension, MAP-OVER-COMMAND-TABLE-KEYSTROKES and MAP-OVER-COMMAND-TABLE-MENU-ITEMS now also accept the keyword argument.

And maybe add a comment marking the keyword parameter as an extension to the function definitions.

Reply:
fixed the commit message according to the suggestion.